packages/react/src/date-range-picker/utils.ts

```diff
@@ -96,3 +97,13 @@ export function getToDateInputButtonAriaLabel(
 	if (!parsed) return 'Choose end date';
 	return `Change end date, ${formatHumanReadableDate(parsed)}`;
 }
+
+/**
+ * Takes a string and converts ISO date strings to a Date object, otherwise converts the string to undefined.
+ */
+export const normaliseDateString = (date: string) => {
```

Comment:
We should move this to date-picker/utils.ts since that's the one that includes the utils used by both date-picker and date-range-picker